src/language/i18n.js

```diff
@@ -16,16 +32,14 @@ i18n.loadLocaleData(defaultLocale, { plurals: () => null })
 i18n.load(defaultLocale, defaultMessages)
 i18n.activate(defaultLocale)
 
-//const isLocaleValid = locale => locales.includes(locale)
+const isLocaleValid = locale => locales.includes(locale)
 const getInitialLocale = () => {
-  // const detectedLocale = detect(fromStorage('lang'), fromNavigator(), () => defaultLocale)
-  // return detectedLocale && isLocaleValid(detectedLocale) ? detectedLocale : defaultLocale
-
-  return defaultLocale
+  const detectedLocale = detect(fromStorage('lang'), fromNavigator(), () => defaultLocale)
```

Comment:
how is this working for native?

Reply:
There is no mention of React Native problems for this function. In theory, the worst case scenario would be using `defaultLocale`. This was commented out because this was not tested on enough different devices with different locales.